rsc/database.properties

```diff
@@ -0,0 +1,3 @@
+jdbc.url=jdbc:mysql://localhost:3306/oh
+jdbc.username=isf
+jdbc.password=isf123
```

Comment:
Since v1.11.0 `.properties` files are not included in the repo, only `.properties.dist`